docs/source/examples/vqe-demo_custom-gate.myst

```diff
@@ -0,0 +1,282 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.3
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Variational Quantum Eigensolver improved with Mitiq
+In this example we investigate how Mitiq can help reduce errors
+and improve convergence of a simple VQE problem executed
+on a simulated noisy backend.
+
+
+The pyQuil VQE example shown here is adapted from
+https://grove-docs.readthedocs.io/en/latest/vqe.html#, 
+and the VQE function in Grove.
```

Comment:
```suggestion
and the VQE function in Grove.
```
Is a verb missing? Like "and the VQE function is taken from Grove"?

Reply:
I realize the introduction could've been clearer. I reworded it, so hopefully it makes more sense now.

Reply:
I reworded this section, so hopefully it's clearer now.